Source/Orts.Simulation/Simulation/RollingStocks/MSTSSteamLocomotive.cs

```diff
@@ -1707,6 +1725,25 @@ public override void Initialize()
                 }
             
             }
+
+            // Check to see if water scoop elements have been configured
+            if (WaterScoopFillElevationM == 0)
+            {
+               WaterScoopFillElevationM = 2.7432f; // Set to default of 9 ft
+            }
+            
+            if (WaterScoopDepthM == 0)
+            {
+               WaterScoopDepthM = 0.0889f; // Set to default of 3.5 ins
+            }
+            
+            if (WaterScoopWidthM == 0)
+            {
+               WaterScoopWidthM = 0.3048f; // Set to default of 1 ft
+            }
```

Comment:
Can these defaults not be done the same way as everything else, in the field initialiser itself? Why must we check for 0 and set it instead of just starting with the default?

Reply:
I have done it this way for parameters that can be overwritten by users through the ENG or WAG files.

In some instances (not currently in this case) an information message is provided in the log to confirm that the user has successfully overwritten the value.

Given that the code is only executed once when OR is initialised, what issues does it create?

Reply:
The main issue is pulling the default values away from where they are for many other things, where people would typically expect to find the defaults, and simply unnecessary code. For example, if the field `WaterScoopWidthM` was initialised with `0.3048f` where it was defined, these four lines would be removed.

I don't understand the point of an informational message to tell the user they have set something, so I'm glad there's none in here.